banks.json

```diff
@@ -34,6 +34,41 @@
     "positiveIndicator": null,
     "negativeIndicator": "-"
   },
+  "bunq-en": {
+    "bankName": "BUNQ-EN",
```

Comment:
This should be the name of the bank in the IBAN. Probably should write that down somewhere.

Please change to `"BUNQ"`

Reply:
Changed.